robot/robot.py

```diff
@@ -25,30 +25,48 @@ def __init__(self, robotd_path=ROBOTD_ADDRESS):
         self.known_servo_boards = []
         self.known_cameras = []
         self.known_gamestates = []
+        print("Initializing Hardware...")
         self.all_known_boards = [
             self.known_power_boards,
             self.known_motor_boards,
             self.known_servo_boards,
             self.known_cameras,
             self.known_gamestates,
         ]
+        self.physical_boards = [
```

Comment:
Unless the "serial" of the `Game` is particularly undesirable (I don't know what it is), I'd encourage just including it in the output. Having _another_ list of lists feels overkill for this functionality.

Reply:
It's not undesirable, I just couldnt think of a good serial number, as the current implementation lacks one. Perhaps just a fixed string would work

Reply:
My reading of the code suggests that `GameState` does indeed have a `.serial`, though it'll end up being the constant value `'state'`.